src/commands/deleteVirtualMachine/promptResourcesToDelete.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IAzureQuickPickItem } from "vscode-azureextensionui";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { ResourceToDelete } from "./deleteConstants";
+
+export async function promptResourcesToDelete(resources: ResourceToDelete[]): Promise<IAzureQuickPickItem<ResourceToDelete>[]> {
+
+    const quickPicks: IAzureQuickPickItem<ResourceToDelete>[] = resources.map(resource => {
+        return { label: resource.resourceName, description: toTitleCase(resource.resourceType), data: resource, picked: resource.picked };
+    });
+
+    return await ext.ui.showQuickPick(quickPicks, { placeHolder: localize('selectResources', 'Select resources to delete'), canPickMany: true });
+}
+
+function toTitleCase(str: string): string {
```

Comment:
In general you just can't do fancy string manipulation like this and keep it localization friendly. Plus I feel like even for english it's a bit questionable ("Ip" vs "IP"). I think they're fine with their existing casing, but if you want the uppercase stuff, you just gotta hard-code a new string for each one